src/content/docs/errors-inbox/apm-tab.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: 'APM: Group errors tab '
+tags:
+  - APM
+  - APM UI pages
+  - Error analytics
+metaDescription: New Relic's error profiles feature in the classic APM view shows trends and anomalies that help you compare and troubleshoot error events.
+redirects:
+  - /docs/apm/applications-menu/error-analytics/error-profiles-troubleshoot-trends
+  - /docs/apm/applications-menu/error-analytics/apm-error-profiles-troubleshoot-trends
+---
+
+import apmGroupErrorsTab from 'images/apm_screenshot-full_group-errors-tab.png'
+
+import apmErrorProfiles from 'images/apm_screenshot-full_error-profiles_2.png'
+
+import apmTriageErrors from 'images/apm_screenshot-full_triage-errors.png'
+
+import apmErrorsOutsideTransactions from 'images/apm_screenshot-full_errors-outside-transactions.png'
+
+The APM group errors tab allows engineering teams to monitor errors in their applications and delve deeply into any issues. 
```

Comment:
Is it the **Group errors** tab? We could do it that format if that's the wording in UI.

Reply:
good catch!